kotlin-dsl/dsl/src/test/kotlin/com/chutneytesting/kotlin/synchronize/ChutneyServerServiceImplTest.kt

```diff
@@ -217,12 +221,188 @@ class ChutneyServerServiceImplTest {
                 assertThat(createRequestReceived.get("content").textValue()).isNotBlank()
                 assertThat(createRequestReceived.get("tags"))
                     .extracting(JsonNode::textValue)
-                    .containsExactlyInAnyOrder(tuple("TAG1"), tuple("TAG2"), tuple("KOTLIN"))
+                    .containsExactlyInAnyOrder(tuple("STAG_1"), tuple("STAG_2"), tuple("TAG1"), tuple("TAG2"))
                 assertThat(createRequestReceived.get("defaultDataset").textValue()).isEqualTo(scenario.defaultDataset)
             }
         }
     }
 
+    @Nested
+    inner class Datasets {
```

Comment:
we could have these in another test class ? May be have sut split in different classes ?